subprojects/plugins/src/integTest/groovy/org/gradle/groovy/compile/LowerToolchainGroovyCompileIntegrationTest.groovy

```diff
@@ -21,6 +21,14 @@ import org.gradle.internal.jvm.Jvm
 
 class LowerToolchainGroovyCompileIntegrationTest extends AbstractToolchainGroovyCompileIntegrationTest {
 
+    @Override
+    def getGroovyJarVariants() {
+        // Do not test with groovy-all with Groovy 4
```

Comment:
Can you add a reason why we don't test with `groovy-all` with Groovy 4?

Reply:
Good point, I will once we have an issue reported to Groovy.

Reply:
This is the issue: https://issues.apache.org/jira/browse/GROOVY-10543

Reply:
Done